app/views/articles/show.html.erb

```diff
@@ -30,13 +30,16 @@
       <%= heading_tag article.title %>
 
       <% unless @newsletter_excluded %>
-        <%= render 'shared/news_signup_test',
-                   ab_test_render: true %>
+        <%= render 'shared/news_signup_test', ab_test_render: true %>
         <%= render 'shared/news_signup_inpage', ab_test_render: false %>
```

Comment:
This isn't part of this story so the removal of ```<%= render 'shared/news_signup_inpage', ab_test_render: false %>``` will be part of another ticket.